app/controllers/groups_controller.rb

```diff
@@ -147,18 +147,9 @@ def group_member_params
   end
 
   def update_leaders
-    return if params[:group][:leader].nil?
-    @group.group_members.each do |group_member|
-      if params[:group][:leader].include? group_member.userid.to_s
-        group_member.update(leader: true)
-        pusher_type = 'add_group_leader'
-      else
-        group_member.update(leader: false)
-        pusher_type = 'remove_group_leader'
-      end
+    updated_leader_ids = params[:group][:leader]
+    return if updated_leader_ids.nil?
 
-      GroupNotifier.new(@group, pusher_type, current_user)
-                   .send_notifications_to(@group.leaders)
```

Comment:
Are the notifications supposed to be sent to the people who were leaders before the changes were made or after?